routes/members.js

```diff
@@ -50,4 +50,6 @@ router.get('/', membersController.getMembers)
 
 router.get('/idle', membersController.getIdleMembers)
 
+router.post('/intro/:username', recruitersController.addRecruiter)
```

Comment:
Can you add the api definition here and generate the schema?
Refer to the other routes and https://github.com/Real-Dev-Squad/website-backend/blob/develop/README.md#api-documentation

Reply:
Added the api definition and generated the schema successfully.